src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/GATKReadFilterPluginArgumentCollection.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.hellbender.cmdline.argumentcollections;
+
+import java.io.Serializable;
+import java.util.Collection;
+
+/**
+ * An abstract ArgumentCollection for specifying parameters for {@link org.broadinstitute.hellbender.cmdline.GATKPlugin.GATKReadFilterPluginDescriptor}.
+ *
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+public abstract class GATKReadFilterPluginArgumentCollection implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    /** Returns the enabled filter names. */
+    public abstract Collection<String> getEnabledReadFilters();
```

Comment:
I don't agree with the typing as a `List`, because some implementations may have a `Set` argument to do not repeat filters.

I agree that they should maintain the ordering, but maybe some clients do not require that in their descriptor because they are not using a composed filter or they are independent of the order.

I added a default implementation returning an empty list.